src/Card/Card.stories.js

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import { Card } from "./Card";
+import { cardAlignOptions } from "./Card.levers";
+import { Heading } from "../Heading/Heading";
+import { CtaLink } from "../CtaLink/CtaLink";
+import { CtaButton } from "../CtaButton/CtaButton";
+
+export default {
+  title: "Composite/Card",
+  component: Card,
+  subcoponent: { Heading, CtaLink, CtaButton },
+  argTypes: {
+    align: {
+      control: {
+        type: "inline-radio",
+        options: cardAlignOptions,
+      },
+    },
+    isMinimal: {
+      control: {
+        type: "boolean",
+      },
+    },
+  },
+};
+
+const CardTemplate = ({ ...rest }) => (
+  <Card {...rest}>
+    {/* Developer would pass in their own card content components */}
+    <span className="su-font-sans su-font-bold su-type-0 su-mb-0">
+      Super headline
+    </span>
+    <Heading level="3" size={1} className="su-mb-0">
+      Headline
+    </Heading>
```

Comment:
It might be redundant now but if we wanted to add a context to the whole card later we might have to wrap this header in another component so that it can `useContext(CardContext)` so it might be best to wrap this in a `<Card.Heading>` sub-component.